components/storage/blockdevices/COMPONENT_SD/README.md

```diff
@@ -0,0 +1,618 @@
+# mbed OS SDCard Driver (sd-driver) for FAT32 Filesystem Support
```

Comment:
That should be moved to the handbook. Having it in both places will result in one or both of them being out of sync.

Reply:
Agreed and I will move them in the next commit.